test/integration/full-content/fixtures/core__freeform.json

```diff
@@ -4,7 +4,7 @@
         "name": "core/freeform",
         "isValid": true,
         "attributes": {
-            "content": "<p>Testing freeform block with some\n</p><div class=\"wp-some-class\">\n\tHTML <span style=\"color: red;\">content</span>\n</div>"
+            "content": "<p>Testing freeform block with some\n<div class=\"wp-some-class\">\n\tHTML <span style=\"color: red;\">content</span>\n</div>"
```

Comment:
This change is interestingly alarming. The markup is obviously busted, but the more interesting question is "Why?" . Why is there an opening `<p>` tag and not a closing? Why is there any `<p>` tag at all (the original content doesn't include it) ?

Reply:
It is added by `autop`, which seems to be missing the trailing `</p>`. Previously `hpq` was adding the paragraph tags, and `autop` left them alone.

I've raised an issue for `autop` here in https://github.com/WordPress/gutenberg/issues/12610

Based on [this](https://github.com/WordPress/gutenberg/blob/master/packages/blocks/src/api/parser.js#L415) I assume that `autop` should be running, so a fix to `autop` will take care of this here.